src/boot/router.js

```diff
@@ -1,8 +1,12 @@
+import { handleHistory } from '../helpers/historyHandler'
+
 export default async ({ Vue, router }) => {
   Vue.prototype.$history = []
+  const history = handleHistory()
```

Comment:
Porque isso virou um _helper_ ao invés de usar o `this.$history` nos lugares?

Reply:
Pq não usar? esta vai ser uma forma que será feita +/- no vue 3 tb, e $history é usada desta forma em outro lugar tb

Reply:
Não disse que não pode usar, só não entendi o porque você criou uma segunda forma sendo que já existia uma convencionada e implementada em outros lugares.

Reply:
A forma já implementada não resolve o nosso problema, porém ela é usada em outro componente que aparentemente funciona, então mantive as 2.